cellprofiler/modules/calculatestatistics.py

```diff
@@ -208,55 +212,63 @@ def add_dose_value(self, can_remove=True):
+log-transform the dose values before fitting a sigmoid curve.
+
+Select *%(NO)s* if your data values indicate only positive vs. negative
+controls.
+''' % globals()))
 
         group.append('wants_save_figure', cps.Binary(
-                '''Create dose/response plots?''', False, doc='''<a name='wants_save_figure'></a>
-            Select <i>%(YES)s</i> if you want to create and save
-            dose response plots. You will be asked for information on how to save the plots.''' % globals()))
+                '''Create dose/response plots?''', False, doc='''Select *%(YES)s* if you want to create and save dose response plots.
+You will be asked for information on how to save the plots.''' % globals()))
 
         group.append('figure_name', cps.Text(
-                "Figure prefix", "", doc='''
-            <i>(Used only when creating dose/response plots)</i><br>
-            CellProfiler will create a file name by appending the measurement name
-            to the prefix you enter here. For instance, if you have objects
-            named, "Cells", the "AreaShape_Area measurement", and a prefix of "Dose_",
-            CellProfiler will save the figure as <i>Dose_Cells_AreaShape_Area.m</i>.
-            Leave this setting blank if you do not want a prefix.'''
+                "Figure prefix", "", doc='''\
+*(Used only when creating dose/response plots)*
+
+CellProfiler will create a file name by appending the measurement name
+to the prefix you enter here. For instance, if you have objects named,
+“Cells”, the “AreaShape_Area measurement”, and a prefix of “Dose_”,
+CellProfiler will save the figure as *Dose_Cells_AreaShape_Area.m*.
+Leave this setting blank if you do not want a prefix.
+'''
         ))
         group.append('pathname', cps.DirectoryPath(
                 "Output file location",
                 dir_choices=[
                     cps.DEFAULT_OUTPUT_FOLDER_NAME, cps.DEFAULT_INPUT_FOLDER_NAME,
                     cps.ABSOLUTE_FOLDER_NAME, cps.DEFAULT_OUTPUT_SUBFOLDER_NAME,
-                    cps.DEFAULT_INPUT_SUBFOLDER_NAME], doc="""
-            <i>(Used only when creating dose/response plots)</i><br>
-            This setting lets you choose the folder for the output
-            files. %(IO_FOLDER_CHOICE_HELP_TEXT)s
-
-            <p>%(IO_WITH_METADATA_HELP_TEXT)s %(USING_METADATA_TAGS_REF)s
-            For instance, if you have a metadata tag named
-            "Plate", you can create a per-plate folder by selecting one of the subfolder options
-            and then specifying the subfolder name as "\g&lt;Plate&gt;". The module will
-            substitute the metadata values for the current image set for any metadata tags in the
-            folder name. %(USING_METADATA_HELP_REF)s.</p>""" % globals()))
+                    cps.DEFAULT_INPUT_SUBFOLDER_NAME], doc="""\
+*(Used only when creating dose/response plots)*
+
+This setting lets you choose the folder for the output files.
+%(IO_FOLDER_CHOICE_HELP_TEXT)s
```

Comment:
The bad formatting here is from importing this from the main help.py file, there's nothing you can do here.